tests/vmi_configuration_test.go

```diff
@@ -617,6 +617,57 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 					&expect.BExp{R: console.RetValue(hexData)},
 				}, 3)).To(Succeed())
 			})
+
+			It("Should configure guest ACPI MSDM with Secret file", func() {
+				const (
+					volumeMsdmSecretName = "volume-msdm-secret"
+					secretWithMsdmName   = "secret-with-msdm-data"
+				)
+				var msdmTable = []byte{
+					0x4d, 0x53, 0x44, 0x4d, 0x24, 0x00, 0x00, 0x00, 0x01, 0x43, 0x43, 0x52,
+					0x41, 0x53, 0x48, 0x20, 0x4d, 0x45, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+					0x88, 0x04, 0x00, 0x00, 0x71, 0x65, 0x6d, 0x75, 0x00, 0x00, 0x00, 0x00,
+				}
+				// To easily compare with console output
+				var hexData string
+				for _, b := range msdmTable {
+					hexData += fmt.Sprintf("%02x", b)
+				}
```

Comment:
Use `hex.EncodeToString`?

Reply:
It does appear to do the same, thanks! I'll give it a try and replace if it works as expected